sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/WrapperSdkExceptionManager.java

```diff
@@ -0,0 +1,139 @@
+package com.microsoft.azure.mobile.crashes;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+
+import com.microsoft.azure.mobile.crashes.ingestion.models.ManagedErrorLog;
+import com.microsoft.azure.mobile.crashes.utils.ErrorLogHelper;
+import com.microsoft.azure.mobile.utils.MobileCenterLog;
+import com.microsoft.azure.mobile.utils.storage.StorageHelper;
+
+import org.json.JSONException;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+
+@SuppressWarnings("WeakerAccess")
+public class WrapperSdkExceptionManager {
+
+    /**
+     * File extension for data files created by this class.
+     */
+    private static final String DATA_FILE_EXTENSION = ".dat";
+
+    /**
+     * Contains wrapper SDK data that has been loaded into memory
+     */
+    private static final Map<String, byte[]> sWrapperExceptionDataContainer = new HashMap<>();
+
+    @VisibleForTesting
+    WrapperSdkExceptionManager() {
+    }
+
+    /**
+     * Store the in-memory wrapper exception data to disk. This should only be used by a wrapper SDK.
+     *
+     * @param data  The data to be saved
+     * @param errorId The associated error UUID
+     */
+    public static void saveWrapperExceptionData(byte[] data, @NonNull UUID errorId) {
+        if (errorId == null) {
```

Comment:
I think lint will complain about this. Either removing `@NonNull` annotation or null-check.